app/assets/javascripts/current_user.coffee

```diff
@@ -0,0 +1,10 @@
+class User
+  constructor: (@data = {}) ->
+
+  get: (param) ->
+    @data[param]
+
+  isSignedIn: ->
+    @get("id")?
+
+@currentUser = App.currentUser = new User(App.currentUserData)
```

Comment:
Could you please clarify what is the preferred what to access `currentUser` -- as global variable or from `App.currentUser`?